server/src/main/java/com/epam/aidial/core/server/util/RegexUtil.java

```diff
@@ -3,16 +3,23 @@
 import lombok.experimental.UtilityClass;
 
 import java.util.ArrayList;
+import java.util.Collection;
 import java.util.Comparator;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 @UtilityClass
 public class RegexUtil {
```

Comment:
Can we move the class to ControllerSelector and change access level to package ?